core/MyOTALogging.cpp

```diff
@@ -22,7 +22,7 @@
 // global variables
 static bool inOTALog = false;
 
-void OTALog(uint8_t logNode, bool enableAck, const char *fmt, ... )
+void OTALog(uint8_t logNode, bool enableEcho, const char *fmt, ... )
```

Comment:
for consistency, better use `echo`

header file also needs to be updated:
https://github.com/mysensors/MySensors/blob/development/core/MyOTALogging.h#L46-L54

Reply:
see initial comment